trainer/persistence/users_inputs_repository.py

```diff
@@ -31,6 +31,15 @@ def get_agent_intent_count(self, agent_name, intent, from_date, to_date):
                 }
             })
 
+    def get_agent_entities_count(self, agent_name, from_date, to_date):
+        return self.users_inputs_repository.aggregate(
```

Comment:
maybe simplify the count logic (like intent) with the risk of not counting same entity many times in a single user input